scripts/docker/zeppelin/bin/env_python_3_with_R.yml

```diff
@@ -1,33 +1,29 @@
-name: base
+name: python_3_with_R
 channels:
   - conda-forge
   - defaults
+  - pyviz
```

Comment:
Why is this channel needed?

Reply:
This is used for installing hvplot (see https://pypi.org/project/hvplot/)

Reply:
hvplot is also available via [conda-forge](https://anaconda.org/conda-forge/hvplot)

Reply:
oh, thanks @Reamer Let me delete it